plugins/aws/ec2/vpcEndpointAcceptance.js

```diff
@@ -0,0 +1,57 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'VPC PrivateLink Endpoint Acceptance Required',
+    category: 'EC2',
+    description: 'Ensures VPC PrivateLink endpoints require acceptance',
+    more_info: 'VPC PrivateLink endpoints should be configured to require acceptance so that access to the endpoint is controlled on a case-by-case basis.',
+    recommended_action: 'Update the VPC PrivateLink endpoint to require acceptance',
+    link: 'https://docs.aws.amazon.com/vpc/latest/userguide/accept-reject-endpoint-requests.html',
+    apis: ['EC2:describeVpcEndpointServices'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        async.each(regions.ec2, function(region, rcb){
+            var describeVpcEndpointServices = helpers.addSource(cache, source,
+                ['ec2', 'describeVpcEndpointServices', region]);
+
+            if (!describeVpcEndpointServices) return rcb();
+
+            if (describeVpcEndpointServices.err || !describeVpcEndpointServices.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for VPC endpoint services: ' + helpers.addError(describeVpcEndpointServices), region);
+                return rcb();
+            }
+
+            if (!describeVpcEndpointServices.data.length) {
+                helpers.addResult(results, 0, 'No VPC endpoint services present', region);
+                return rcb();
+            }
+
+            for (var s in describeVpcEndpointServices.data) {
+                var service = describeVpcEndpointServices.data[s];
+
+                if (service.Owner === 'amazon') continue;
```

Comment:
This `continue` is okay, but the issue is that if _all_ of the endpoints are owned by `amazon` then we will have no results because everything was skipped (every plugin should produce at least 1 result per region).

To fix this, you can either filter our the array before doing the loop (and check for array.length). Or, you can set a variable like:
```
var found = false;
```
and then, if the code makes it to line 39, `found = true`

and at the end, check `if (!found) helpers.addResult(results, 0, 'No user-owned endpoints found'...`